### PR TITLE
Add Controllers

### PR DESCRIPTION
**Pull request by SaboorSohaib**
### Hello :wave: 

In this milestone, I implemented the following requirements:

- [ ]  For each URL you should create:

   :diamond_shape_with_a_dot_inside:  A route.
   :diamond_shape_with_a_dot_inside:  Action in the correct controller.
   :diamond_shape_with_a_dot_inside:  A view file.

- [ ] View files should include only placeholders written in plain HTML, e.g:

   :diamond_shape_with_a_dot_inside:  Here is a list of posts for a given user
   :diamond_shape_with_a_dot_inside:  Make sure that you added code to handle only the 4 required URLs.

**Kindly check my PR.**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,14 @@
 Rails.application.routes.draw do
+  # get 'users/index'
+  # get 'users/show'
+  # get 'users/posts'
+  # get 'posts/index'
+  # get 'posts/show'
```
Comment (julie-ify) [inline config/routes.rb]: - [ ] **[OPTIONAL]** I would recommend you remove these commented routes.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @SaboorSohaib,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
- [ ] Kindly fix your project title.

  image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
Hi @SaboorSohaib,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (SaboorSohaib) [reply config/routes.rb]: ### Hi @julie-ify 
Thank you very much for reviewing my project.

**Pull request merged**